etor/src/main/java/gov/hhs/cdc/trustedintermediary/external/database/DatabasePartnerMetadataStorage.java

```diff
@@ -197,8 +195,16 @@ private List<DbColumn> createDbColumnsFromMetadata(PartnerMetadata metadata)
                         false,
                         Types.VARCHAR),
                 new DbColumn("sent_message_id", metadata.sentSubmissionId(), true, Types.VARCHAR),
-                new DbColumn("sender", metadata.sender(), false, Types.VARCHAR),
-                new DbColumn("receiver", metadata.receiver(), true, Types.VARCHAR),
+                new DbColumn(
+                        "sender",
+                        metadata.sendingFacilityDetails().namespace(),
```

Comment:
I don't think the namespace is the right thing to store here

Reply:
yes